src/woo_publications/publications/api/filters.py

```diff
@@ -226,3 +226,18 @@ def search_official_and_short_title(self, queryset, name: str, value: str):
         return queryset.filter(
             Q(officiele_titel__icontains=value) | Q(verkorte_titel__icontains=value)
         )
+
+
+class TopicFilterSet(FilterSet):
```

Comment:
not specified, but we'll need a filter field on publicatiestatus as well so that GPP-burgerportaal can request only the published topics.

I've considered filtering this in the viewset, but that prevents GPP-app from working with non-published topics.